test/Validator/PhoneNumberTest.php

```diff
@@ -3164,10 +3164,12 @@ public function testAllowPossibleSetterGetter()
         $this->assertTrue($this->validator->allowPossible());
     }
 
-    public function testSetCountryMethodIsCaseInsensitive()
```

Comment:
I know changing tests is not good but maybe not a BC break? Anyway `getCountry()` is used only in one place and I covered it so maybe not a problem?